### ISSUE TITLE
Status/future of CSRMC drivers

### ISSUE THREAD

**Issue by jtkrogel** [QMCPACK/qmcpack#819]
I noticed that the following files are not integrated with the mainline distribution in any way:
```
QMCDrivers/CorrelatedSampling/CSRMC.h
QMCDrivers/CorrelatedSampling/CSRMCUpdateAll.h
QMCDrivers/CorrelatedSampling/CSRMCUpdatePbyP.h
```

Are there any plans for these files?  If there are then we should keep them and otherwise remove them.

**Comment by rcclay** [comment]
We should remove them.  They'll be added if/when they are implemented.

**Comment by jtkrogel** [comment]
OK, will remove.

**Comment by jtkrogel** [comment]
Resolved by #949
